src/parser/show_meta.c

```diff
@@ -445,6 +445,9 @@ metadata_of_index_capacity (SHOW_ONLY_ALL flag)
     {"Btid", "varchar(64)"},
     {"Num_distinct_key", "int"},
     {"Total_value", "bigint"},
+#if defined(SUPPORT_DEDUPLICATE_KEY_MODE)
+    {"Deduplicate_distinct_key", "int"},
```

Comment:
I think 'Deduplicate_distinct_key' should be output right after 'Num_distinct_key'.

And this may be trivial, but 'Deduplicate_distinct_key' does not contain the word 'Num' .
Consider changing the names for consistency with other columns.

e.g.
  - Num_deduplicate_distinct_key
  - Num_distinct_key_with_deduplicate

Reply:
The order of exposure is at the request of MyungHwan Oh.